app/services/chf/create_derivatives_on_s3_service.rb

```diff
@@ -301,45 +301,78 @@ def create_image_jpg_derivative_future(width:, type_key:)
       end
     end
 
-
-    # Takes a pdf, makes a thumb. Includes small border around thumb, since this
-    # works better for typical white-bg print-like PDFs.
-    def create_pdf_jpg_thumb_future(width:, type_key:)
+    def create_pdf_thumb_future(width:, type_key:)
       defn = get_type_defn(type_key)
```

Comment:
any reason you changed the name from `create_pdf_jpg_thumb_future`?  I had `jpg` in there to indicate that the output is a "jpg", and because other similar methods in this case include "jpg" in their names for same.

Reply:
Nah, this is just an artifact. Will change back.